public/app/features/plugins/extensions/registry/AddedComponentsRegistry.ts

```diff
@@ -52,7 +60,7 @@ export class AddedComponentsRegistry extends Registry<AddedComponentRegistryItem
           continue;
         }
 
-        if (!extensionPointEndsWithVersion(extensionPointId)) {
+        if (!isGrafanaCoreExtensionPoint(extensionPointId) && !extensionPointEndsWithVersion(extensionPointId)) {
```

Comment:
Let's skip this warning for [extension points defined in core grafana ](https://grafana.com/developers/plugin-tools/reference/ui-extensions#extension-points-in-grafana).